mopidy/mpd/__init__.py

```diff
@@ -19,7 +19,7 @@ def get_default_config(self):
     def get_config_schema(self):
         schema = super(Extension, self).get_config_schema()
         schema['hostname'] = config.Hostname()
-        schema['port'] = config.Port()
+        schema['port'] = config.Port(optional=True)
```

Comment:
Yeah I suppose not. I'll add tests and remove the Zeroconf related stuff if using a unix socket.  
Is there any notion of a config being one of two values? Eg.
```python
schema['hostname'] = config.Hostname() or config.UnixSocket()
```
If not, should there be or should I just wrap both of these into the Hostname class?